BZ-2087151: Added note about specifying a priority class name for hos… (#49545)
openshift/openshift-docs: modules/nodes-descheduler-installing.adoc

By default, the descheduler runs in predictive mode, which means that it only simulates pod evictions. You must change the mode to automatic for the descheduler to perform the pod evictions.

[IMPORTANT]
====

> If you have enabled hosted control planes in your cluster, set a custom priority threshold to ensure that pods in the hosted control plane namespaces do not get evicted. Set the priority threshold class name to `hypershift-control-plane`, since it has the lowest priority value (`100000000`) of the hosted control plane priority classes.

Reviewer: Setting a priority class will not always protect a pod from eviction. The higher the priority class the lower the chance of getting evicted. By default only pods with system-critical priority classes are guaranteed to be excluded from eviction.

Author: @ingvagabund So what are you suggesting that we change this to? Something like this?

```suggestion
If you have enabled hosted control planes in your cluster, set a custom priority threshold to lower the chance that pods in the hosted control plane namespaces are evicted. Set the priority threshold class name to `hypershift-control-plane`, since it has the lowest priority value (`100000000`) of the hosted control plane priority classes.
```

> If you have enabled hosted control planes in your cluster, set a custom priority threshold to lower the chance that pods in the hosted control plane namespaces are evicted. Set the priority threshold class name to `hypershift-control-plane`, since it has the lowest priority value (`100000000`) of the hosted control plane priority classes.

Reviewer 2: s/since/because per ISG
